test/src/main/java/org/apache/accumulo/test/ComprehensiveIT.java

```diff
@@ -133,6 +152,61 @@ public static void teardown() {
     SharedMiniClusterBase.stopMiniCluster();
   }
 
+  @Test
+  public void testEventualScan() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+
+      String table = getUniqueNames(1)[0];
+      client.tableOperations().create(table);
+
+      // Start Scan Server
+      getCluster().getClusterControl().start(ServerType.SCAN_SERVER);
+
+      // generate 100 rows of data
+      write(client, table, generateMutations(0, 100, tr -> true));
+
+      // need flush table so that eventual scan can see it.
+
+      long minWait = 5_000;
+      var startTime = System.currentTimeMillis();
+      client.tableOperations().flush(table, null, null, true);
+      var duration = System.currentTimeMillis() - startTime;
+      // if duration is less than minWait, wait the remainder + 0.5 seconds.
+      if (minWait > duration) {
+        Thread.sleep((minWait - duration) + 500);
+      }
```

Comment:
There is no need to sleep here if we assume nothing else is using the scan servers, which is the case for this test.  The first scan will cause the scan server to read and cache the tablets files.

```suggestion
      client.tableOperations().flush(table, null, null, true);
```